src/components/App.tsx

```diff
@@ -262,7 +263,29 @@ export class App extends React.Component<any, AppState> {
                 sceneAppState || getDefaultAppState(),
                 { scrollToContent: true },
               );
-              elements = restoredState.elements;
+              if (elements == null || elements.length === 0) {
+                elements = restoredState.elements;
+              } else {
```

Comment:
Could you add a comment here saying what’s happening here. This is not immediately obvious when reading the code